### DISCUSSION TITLE
Is there a way to convert 16-bit color HW-accelerated D3D games to 32-bit?

### DISCUSSION THREAD

**Discussion by DSN9990** [JHRobotics/softgpu#78] [Q&A]
Hello again, I would like to have an registry setting to convert 16-bit color games to 32-bit such as Sega Rally 2, Forsaken, etc.

This is due to the fact that these games do not natively support 32-bit color.

Is that possible? If not, can you add an registry setting to do this similar to Glide games (which already has an 16-bit to 32-bit renderer)?

**Comment by DSN9990** [comment]
Hi @JHRobotics, can you help me with some issues in two Win9x games I have, please?

**Comment by hughobrien** [comment]
I apologise to JH if this is out of line, but @DSN9990 I think that you should stop trying to create urgency around your questions. It is not in the spirit of the open source work happening here.

Asking questions is great and encouraged, but chasing up for answers for a volunteer project is in my opinion rude. Patience is required.

**Reply by DSN9990** [reply]
And yeah, sorry for being impatient again.

**Reply by DSN9990** [reply]
I really love softgpu since I discovered it, since it is a nice addition to VMWare Workstation because this virtualizer does not support 3D acceleration officially on Windows 98 in any way. I hope in the future it will be a lot better for gaming in Win9x.

**Comment by kjliew** [comment]
@DSN9990 How about a small donation to **Game Preservation** 😁? Many games that you have reported here, work & play well, such as:

* Uprising 2: Lead and Destroy
* Half-Life 1
* SEGA Rally 2
* Dirt Track Racing - Sprint Cars
* Dirt Track Racing 2
* Leadfoot Stadium Off-Road Racing

**Comment by DSN9990** [comment]
> @DSN9990 How about a small donation to **Game Preservation** 😁? Many games that you have reported here, work & play well, such as:
> 
> * Uprising 2: Lead and Destroy
> * Half-Life 1
> * SEGA Rally 2
> * Dirt Track Racing - Sprint Cars
> * Dirt Track Racing 2
> * Leadfoot Stadium Off-Road Racing

No, thanks. I prefer softgpu because it supports DirectX up to 9.0c, and it is faster than QEMU-3dfx on my pc.

**Comment by DSN9990** [comment]
Hello all. Anyone knows JHRobotics is doing at the moment? He is in hiatus since the last month.

**Reply by JHRobotics** [reply]
Sorry, my free time was in last 2 months very limited. I’m currently browsing through issues and pushing code/changes. Some problems will be solved in next release (ETA end of June/begin of July) but some are in VirtualBox itself and I have to submit these patches to Oracle and hope that they will merge it soon.

**Reply by DSN9990** [reply]
> Sorry, my free time was in last 2 months very limited. I’m currently browsing through issues and pushing code/changes. Some problems will be solved in next release (ETA end of June/begin of July) but some are in VirtualBox itself and I have to submit these patches to Oracle and hope that they will merge it soon.

Then, when will the next version be released?

**Reply by JHRobotics** [reply]
Exactly when DirectX applications in vmware will stop looking like this:

https://github.com/user-attachments/assets/6625b84b-3b9f-41a6-a6f3-da015465828c

**Reply by JHRobotics** [reply]
New release is out now! :-)
